sdk/storage/Cargo.toml

```diff
@@ -19,40 +19,184 @@ base64 = "0.13"
 chrono = "0.4"
 http = "0.2"
 futures = "0.3"
-hyper = "0.14"
 log = "0.4"
-mime = "0.3"
 quick-error = "1.2"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
 serde-xml-rs = "0.4"
-time = "0.2"
 url = "2.2"
 uuid = { version = "0.8", features = ["v4"] }
-smallvec = { version = "1.6", features = ["serde"] }
 bytes = "1.0"
-hyper-rustls = "0.22"
 md5 = "0.7"
 RustyXML = "0.3"
-percent-encoding = "2.1"
-failure = "0.1"
 thiserror = "1.0"
 once_cell = "1.7"
 
 [dev-dependencies]
 tokio = { version = "1.0", features = ["macros"] }
 env_logger = "0.8"
 azure_identity = { path = "../identity" }
-url = "2.2"
 reqwest = "0.11"
 
 [features]
-default = ["account", "blob", "queue", "table", "data_lake"]
-test_e2e = ["account", "blob", "queue", "table", "data_lake"]
-account = []
+default = ["storage_account", "blob_storage", "queue_storage", "table_storage", "data_lake_storage"]
+test_e2e = ["storage_account", "blob_storage", "queue_storage", "table_storage", "data_lake_storage"]
+storage_account = []
 azurite_workaround = []
-blob = []
-queue = []
-table = []
-data_lake = []
+blob_storage = []
+queue_storage = []
+table_storage = []
+data_lake_storage = []
+
+
+
+# This section specifies the required features for the examples. 
+
+[[example]]
+name="table_00"
+required-features = ["table_storage"]
+
```

Comment:
Can we remove all the blank spaces?

Reply:
I did it to separate the different example groups. Will do!